### DISCUSSION TITLE
Authenticator.cc Help

### DISCUSSION THREAD

**Discussion by kaiwilliams-dev** [Authenticator-Extension/Authenticator#1151] [General]
Hey everyone, I hope you're all well. I activated 2-factor authentication for Facebook account using Authenticator.cc. Now that plugin is not working and I cannot get into my Facebook account. Meta has asked me to submit my ID and video of myself and I have done that 3 times and still nothing. So the problem is that I can not access my Facebook account and all my assets using my login details because of that 2-factor authentication, I do not get a code when I try to log in, and I have not changed any of my accounts. Please help
![Screenshot 2024-02-09 at 12-52-13 Business Support Home](https://github.com/Authenticator-Extension/Authenticator/assets/159545283/b7f20aa0-eb65-4f6a-96e7-236d76c5208f)
![Screenshot 2024-02-09 at 15-08-05 mymindstorm (Brendan Early) · GitHub](https://github.com/Authenticator-Extension/Authenticator/assets/159545283/6886be17-3ed1-491c-915a-46abc9e6fcea)

**Comment by kaiwilliams-dev** [comment]
![Screenshot 2024-02-09 at 15-30-22 Business Support Home](https://github.com/Authenticator-Extension/Authenticator/assets/159545283/dca04f5b-0a00-4f7d-b3ef-1b102bfe38d3)

**Comment by mymindstorm** [comment]
What do you mean by not working? Please be more specific as to what the error is.

**Comment by kaiwilliams-dev** [comment]
Hey Brendan, thank you for getting back to me, I had registered the plug-in
to give me a code for my Facebook login, but on Monday when I tried to
login to Facebook it was somehow disconnected and not giving me code

**Reply by mymindstorm** [reply]
If there is a missing code, the only way to get back to your account is to restore from a backup. Otherwise you need to ask Facebook to restore your account using https://m.facebook.com/help/147926301947841

**Comment by kaiwilliams-dev** [comment]
I have tried, please let me get back to you if I am successful. Thank you
so much for helping
